app/src/main/java/com/gathering/android/auth/password/ResetPasswordInfo.kt

```diff
@@ -0,0 +1,122 @@
+package com.gathering.android.auth.password
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.annotation.StringRes
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.wrapContentSize
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Surface
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.platform.ComposeView
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.FontStyle
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.fragment.app.DialogFragment
+import com.gathering.android.R
+import com.gathering.android.common.CustomTextView
+import com.gathering.android.common.isComposeEnabled
+import com.gathering.android.databinding.BottomSheetForgetPassInfoBinding
+import com.gathering.android.ui.theme.GatheringTheme
+import com.google.android.material.bottomsheet.BottomSheetDialogFragment
+import dagger.hilt.android.AndroidEntryPoint
+
+@AndroidEntryPoint
+class ResetPasswordInfo : BottomSheetDialogFragment() {
+
+    private lateinit var binding: BottomSheetForgetPassInfoBinding
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setStyle(
```

Comment:
should this also be set in case compose is enabled ?

Reply:
we have never touched this part of the code in other classes

Reply:
considering we have a GatheringTheme it could lead to unpredicted behaviour. i for one am not sure how legacy styles work with compose themes. which one takes precedence ?

please read on this more and create a ticket if there is a compose alternative for it

Reply:
so short answer: no
view with compose has nothing to do with this onCreate method and its contents.

Reply:
well this block of code (onCreate) has nothing to do with gatheringTheme or compose view! like you said that's only for when featureFlaag is off(false) and in that case this fragment will use the onCreate method to style the bottomsheet.

Reply:
i will get more into the themes and styles as the theme we have right now is just a basic one and following compose best practices. i will look into it more and have a more customized theme later on.

Reply:
how is that for feature flag off if there is no conditional block there ?

Reply:
the safest way for doing this would be wrapping set style in a `if(!composeEnabled)`

Reply:
ok i will add that right away1

Reply:
is what you are suggesting to do , exception handling? @amirziarati

Reply:
i would like to do that for all other classes that I have converted to compose so far and I would like to create a separate ticket for that and have you take a look after it's done?

Reply:
@amirziarati 

https://github.com/moyarsuccess/gathering/pull/92